src/addons/dragAndDrop/EventContainerWrapper.js

```diff
@@ -157,7 +184,11 @@ class EventContainerWrapper extends React.Component {
       const bounds = getBoundsForNode(node)
       const { dragAndDropAction } = this.context.draggable
 
-      if (dragAndDropAction.action === 'move') this.handleMove(box, bounds)
+      if (dragAndDropAction.action === 'move') {
+        if (this.context.draggable.enableDragAutoScroll)
+          this.updateParentScroll(parent, node)
+        this.handleMove(box, bounds)
+      }
```

Comment:
Though this logic seems reasonable here, if we were to emit a onDragging event as per: https://github.com/jquense/react-big-calendar/pull/2061/files it would be good to call updateParentScroll inside handleMove function. If onDragging returns false, we can choose not to call updateParentScroll.

Reply:
As of now, it will be fine since we need `parent` and `node` params for `updateParentScroll`